### DISCUSSION TITLE
What is the difference between "-statcell group"(-T) and "-statelset group"(-M)?

### DISCUSSION THREAD

**Discussion by smallrain1996** [neperfepx/neper#690] [Q&A]
As my title suggests, my test code is as follows. The content in group1.stelset is all -1, which I cannot understand.

```
mkdir test
cd test
neper -T -n 100 -group "id<=50?1:2" -statcell group -o group1
neper -M group1.tess -statelset group -format inp -elttype hex -o group1
```
I think the content of group1.stelset and group1.stcell in my test example should be the same. I wonder if there is something wrong with my understanding.

**Comment by rquey** [comment]
Your understanding is correct. It was working for tet meshing but not hex meshing.  This is now fixed (version `4.6.1-2`). The values in the stcell and stelset files will now match.
